source/multiple-mongosyncs.txt

```diff
@@ -0,0 +1,266 @@
+
+.. code-block:: shell
+
+   curl monogsync01Host:27601/api/v1/start -XPOST --data \
+        '{ "source": "cluster0", "destination": "cluster1", \
+           "reversible": false, "enableUserWriteBlocking": false }'
+
+   curl monogsync02Host:27602/api/v1/start -XPOST --data \
+        '{ "source": "cluster0", "destination": "cluster1", \
+           "reversible": false, "enableUserWriteBlocking": false }'
+
+The ``start`` command options must be the same for all of the ``mongosync``
+instances.
+
+.. _c2c-sharded-progress:
+
+Check Progress
+--------------
+
+The ``mongosync`` instances do not aggregate progress information
+across shards. To review synchronization progress for a particular
+shard, use ``curl`` or another HTTP client to issue the
+:ref:`progress <c2c-api-progress>` command to the ``mongosync``
+instance synchronizing that shard.
+
+.. code-block:: shell
+
+   curl monogsync02Host:27602/api/v1/progress -XGET
+
+This command checks the progress of the ``mongosync`` instance that is
+running on ``monogsync02Host`` and using ``port 27602`` for
+synchronization. To check progress on other shards, update the host and
+port number then repeat the API call to each ``mongosync`` instance.
+
+.. _c2c-sharded-pause:
+
+Pause a ``mongosync`` Instance
+------------------------------
+
+The :ref:`pause <c2c-api-pause>` command will temporarily halt the
+synchronization process on a single shard. It does not pause any other
+``mongosync`` instances that may be running. Use ``curl`` or another
+HTTP client to issue the ``pause`` command to a ``mongosync`` instance.
+
+.. code-block:: shell
+
+   curl monogsync01Host:27601/api/v1/pause -XPOST --data '{}'
+
+This command pauses the ``mongosync`` instance that is running on
+``monogsync01Host`` and using ``port 27601`` for synchronization. To
+pause synchronization on other shards, update the host and port number
+then repeat the API call to each ``mongosync`` instance.
+
+.. _c2c-sharded-resume:
+
+Resume Synchronization
+----------------------
+
+If one or more ``mongosync`` instances are paused, you can use the
+:ref:`resume <c2c-api-resume>` command to resume synchronizing. Run a
+separate ``resume`` command against each paused ``mongosync`` instance
+to continue synchronizing.
+
+Use ``curl`` or another HTTP client to issue the :ref:`resume
+<c2c-api-resume>` command to each ``mongosync`` instance.
+
+.. code-block:: shell
+
+   curl monogsync01Host:27601/api/v1/resume -XPOST --data '{}'
+
+This command resumes synchronization on the ``mongosync`` instance that
+is running on ``monogsync01Host`` and using ``port 27601``.  To 
+resume synchronization on other shards, update the host and port number
+then repeat the API call to each ``mongosync`` instance.
+
+.. _c2c-sharded-commit:
+
+Commit Synchronization From  Multiple ``mongosync`` Instances
+-------------------------------------------------------------
+
+When synchronization is complete, issue the :ref:`commit
```

Comment:
just a question about this phrasing, not sure what the language is elsewhere in the docs but my understanding is that issuing the commit command is what completes the synchronization, and before issuing the commit command the synchronization is still ongoing, so what does "when synchronization is complete" mean in these docs?

Reply:
@huayu-ouyang  I think this language came from the spec docs. 

@ldennis What does it mean for synchronization to be complete (pre-commit)?  Maybe another way to ask is, how can a user determine that it is time to send the /commit call to mongosync?

Reply:
Sorry, I think I missed that. I think it is totally up to the users when to issue /commit. They technically can commit whenever canCommit: true from the progress API. But normally, we would recommend issuing the commit API only when the lag time is small enough for their use case.

Reply:
Maybe instead of saying "when synchronization is complete" we could say something like "When you want to complete synchronization, check that canCommit: true, and then issue the commit command to each mongosync instance"? and we could add a note that we recommend only issuing the commit command when the lag time is small enough for their use case?